src/content/topics/integrations/observability/chronosphere.mdx

```diff
@@ -0,0 +1,74 @@
+---
+path: /integrations/chronosphere
+title: Chronosphere
+description: This topic explains how to configure and send LaunchDarkly events to
+Chronosphere.
```

Comment:
```suggestion
description: This topic explains how to configure and send LaunchDarkly events to Chronosphere.
```
Our frontmatter formatting is picky; having the description on two lines was causing the page not to render.

Reply:
Ah, thanks for that callout! I'll put the `description` on a single line.